os/board/rtl8720e/src/component/bluetooth/os/tizenrt/osif_tizenrt.c

```diff
@@ -617,7 +617,7 @@ bool osif_msg_peek(void *p_handle, void *p_msg, uint32_t wait_ms)
 /****************************************************************************/
 void *osif_mem_alloc(RAM_TYPE ram_type, size_t size)
 {
-	return kmm_malloc(size);
+	return rtw_malloc(size);
```

Comment:
Is this function wrapping rtw_malloc()?
If re-wrapping the warpped function of malloc API, we need to add DEBUG_SET_CALLER_ADDR() macro.

This function(osif_mem_alloc()) seems to be called from many location.
so I think, we need this for tracking who is actually asking alloc.

Reply:
You are right, I have modified it.